.travis.yml

```diff
@@ -30,7 +30,7 @@ before_install:
     TARGET_PKG: cherry_picker
   install:
   - &cd-to-project pushd "$TARGET_PKG"
-  - flit install
+  - &flit-install flit install --extras dev,test
```

Comment:
The command is 
`flit install --deps develop`

It will install dev and tests requirements.

Reply:
I tried this: see ~10 last commits :(

Reply:
I tried this: see ~10 last commits :(